web/packages/teleport/src/Console/ActionBar/ActionBar.tsx

```diff
@@ -34,20 +30,6 @@ export default function ActionBar(props: Props) {
       {props.latencyIndicator.isVisible && (
         <LatencyDiagnostic latency={props.latencyIndicator.latency} />
       )}
-      <MenuIcon
-        buttonIconProps={{ mr: 2, ml: 2, size: 0, style: { fontSize: '16px' } }}
-        menuProps={menuProps}
-      >
-        <MenuItem as={NavLink} to={cfg.routes.root}>
-          <MenuItemIcon as={Icons.Home} mr="2" size="medium" />
-          Home
-        </MenuItem>
-        <MenuItem>
-          <ButtonPrimary my={3} block onClick={props.onLogout}>
-            Sign Out
```

Comment:
I'm not sure about removing the logout button from the security perspective: 
https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#logout-button
With this change, the user will no longer be able to log out from the "console" screen, and even have to open a new tab with Teleport to log out from there 🤔

>and we've seen some users mistakenly click the sign out button and terminate all of their sessions.

So maybe we should make it more clear, like `Sign Out of Teleport` (and convert to a list):

<img width="456" alt="image" src="https://github.com/user-attachments/assets/8f5f36d1-de2a-486f-a40d-9ba3e309f3dd">